buổi 10/main.js

```diff
@@ -0,0 +1,98 @@
+// Bài 1:
+// Cách 1:
+const getArray = (string, lengthArray) => {
+  const arr = []
+  for(let i = 0; i < lengthArray; i++ ) {
+      arr.push(string)
+  }
+  return arr
+}
+
+//Cách 2:
+const getArray = (string, lengthArray) => {
+	let i = 0 
+	const arr = []
+	do {
+		arr.push(string)
+		i++
+	} while(i < lengthArray)
+	return arr
+}
+
+//Bài 2:
```

Comment:
```suggestion
// Bài 2:
```
Và không được dùng tiếng Việt